features/cucumber.feature

```diff
@@ -19,7 +19,6 @@ Feature: The cucumber command
     Then the output should contain "# Running features"
       And the output should contain "> Only: features/single.feature"
       And the output should contain "Features green."
-    But the output should not contain "> Using parallel_tests"
```

Comment:
I believe this cannot be removed. Please revert to the previous `should not contain "parallel"`.